main.c

```diff
@@ -1,20 +1,19 @@
-#include <stdio.h>
 #include "lua.h"
 #include <lauxlib.h>
 #include <lualib.h>
 
 int run_lua(const char* script) {
-	lua_State* lua = luaL_newstate();
-	luaL_openlibs(lua);
+	size_t len;
+	const char* value;
+	lua_State* L = luaL_newstate();
```

Comment:
Just a small style nit but I'd prefer to keep uppercase for constants and the like.